packages/ui/.eslintrc.js

```diff
@@ -7,5 +7,7 @@ module.exports = {
   ignorePatterns: ['dist/**/*'],
   rules: {
     'import/no-extraneous-dependencies': 'off',
+    'import/no-default-export': 'off',
+    'react/jsx-props-no-spreading': 'off',
```

Comment:
```suggestion
```

Let add the stories into the ignore pattern instead.
I prefer having clean components that they follow the same patters and leave the stories flexible to use all the techniques we want. Stories is not code users will use in production in the end

Reply:
Great solution! I keep forgetting about the `ignorePatters`! I hope this is the last time I forget about it! Thanks man